RatScanner/View/BlazorUI.xaml.cs

```diff
@@ -42,16 +42,6 @@ public BlazorUI()
 		IconScanHotkeySelector.Height = 0;
 		serviceCollection.AddSingleton<IHotkeySelector>(s => IconScanHotkeySelector);
 
-		var bounds = System.Windows.Forms.Screen.AllScreens.Select(screen => screen.Bounds);
```

Comment:
You probably already tried this but might there be a dpi property or similar hidden behind `System.Windows.Forms.Screen.AllScreens` which could be used instead of the dark magic in `ScreenInfo`?

Reply:
Unfortunately there is not